src/runtime/sema.go

```diff
@@ -373,19 +373,11 @@ Found:
 func (root *semaRoot) rotateLeft(x *sudog) {
 	// p -> (x a (y b c))
 	p := x.parent
-	a, y := x.prev, x.next
```

Comment:
I checked the whole file but the code not seems to be unnecessary. Even though, I think the code remain here is to exemplify what was the logic here before they code this?